app/models/aggregations/materialized_view.rb

```diff
@@ -1,4 +1,6 @@
-class Aggregations::MaterializedView
+class Aggregations::MaterializedView < ActiveRecord::Base
```

Comment:
We should only inherit from `ApplicationRecord` as it's Rails standards - see https://blog.bigbinary.com/2015/12/28/application-record-in-rails-5.html

Reply:
The MaterializeView acts as the "views" ApplicationRecord.